### PR TITLE
Update rubygem-gettext to remove .po and samples folders

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/rubygem-gettext/rubygem-gettext.spec [hunk 1]
```diff
@@ -85,8 +85,6 @@ find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 %{gem_instdir}/bin
 %{gem_libdir}
 %{gem_instdir}/locale
-%{gem_instdir}/po
```
Comment (ekohl) [inline packages/foreman/rubygem-gettext/rubygem-gettext.spec:88]: Will this pass or do you need to use `%exclude`?

Comment (zjhuntin) [reply packages/foreman/rubygem-gettext/rubygem-gettext.spec:88]: probably need to exclude

**Comment by ehelms** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Pull request merged**